uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -49,20 +50,18 @@ fun AppContent(
     val deepLinkUri =
         "https://" + LocalContext.current.getString(R.string.deep_link_host) +
             LocalContext.current.getString(R.string.deep_link_path)
-    val actions = remember(navController) { AppActions(navController) }
+    val actions = remember(navController, drawerContentState) {
```

Comment:
I have a question! Do you need this key? 🙋

Reply:
Sorry for missing it. I'll fix it now. :smile: